README.md

```diff
@@ -16,151 +16,17 @@ Run `make run` to start the operator.
    `IMG=<image repository and tag> make docker-build docker-push` 
    
    for example: `IMG=quay.io/jdzon/k4e-operator:latest make docker-build docker-push`
-   
- - Deploy the operator:
-   
-   `IMG=<image repository and tag> make deploy`
-
-### Port forwarding
```

Comment:
Is there a place where we would like to add info about port forwarding?

Reply:
there shouldn't be a need for it after we merge https://github.com/jakub-dzon/k4e-operator/pull/63 since the service will be exposed either via the ingress or route.